command/ca/sign.go

```diff
@@ -82,6 +83,12 @@ $ step ca sign foo.csr foo.crt \
 --acme https://acme-staging-v02.api.letsencrypt.org/directory
 '''`,
 		Flags: []cli.Flag{
+			acmeFlag,
+			acmeContactFlag,
+			acmeHTTPListenFlag,
+			acmeStandaloneFlag,
+			acmeWebrootFlag,
+			consoleFlag,
```

Comment:
I don't think acme and console flags are the most important, move them down.

Reply:
So, I didn't realize this, but it seems the order here is reflected as the order in the help doc. That's rather unfortunate, as it makes the flag array pretty difficult to reason about, given our mix of global, local, and in the moment defined flags. I down for updating this, but it would be great if we came up with something consistent.

Reply:
moved these around to a more appropriate order